packages/dataviews/src/view-actions.js

```diff
@@ -3,24 +3,23 @@
  */
 import {
 	Button,
-	Icon,
 	privateApis as componentsPrivateApis,
 } from '@wordpress/components';
-import { chevronRightSmall, check, arrowUp, arrowDown } from '@wordpress/icons';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { unlock } from './lock-unlock';
-import { VIEW_LAYOUTS, LAYOUT_TABLE } from './constants';
+import { VIEW_LAYOUTS, LAYOUT_TABLE, SORTING_DIRECTIONS } from './constants';
+import { DropdownMenuRadioItemCustom } from './dropdown-menu-helper';
```

Comment:
@ciampo do we have a v2 radio item we can use here? As far as my understanding goes, we only need a custom radio for cases where we need to be able to uncheck a checked option (only filters that I'm aware of). It's not the case in any of these components.

Reply:
See https://github.com/WordPress/gutenberg/pull/55625#discussion_r1440307212